staging/src/k8s.io/kubectl/pkg/describe/describe_test.go

```diff
@@ -875,9 +953,9 @@ func TestDescribeService(t *testing.T) {
 						},
 					},
 					Endpoints: []discoveryv1.Endpoint{
-						{Addresses: []string{"10.244.0.3"}},
-						{Addresses: []string{"10.244.0.4"}},
-						{Addresses: []string{"10.244.0.5"}},
+						{Addresses: []string{"10.244.0.3"}, Conditions: discoveryv1.EndpointConditions{Ready: ptr.To(true)}},
+						{Addresses: []string{"10.244.0.4"}, Conditions: discoveryv1.EndpointConditions{Ready: ptr.To(true)}},
```

Comment:
nit: can we have a ready=false condition in any of the endpoint here for assertion

Reply:
> nit: can we have a ready=false condition in any of the endpoint here for assertion

yes, indeed needed, and it also find a bug.